<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalProfiles</class>
 <widget class="QWidget" name="PersonalProfiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1952</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="selectEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selectLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Profile:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>540</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>540</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="basalLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Basal Rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="correctionLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Correction Factor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="carbsLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Carbohydrates Ratio:</string>
   </property>
  </widget>
  <widget class="QLabel" name="targetLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Target Glucose Levels:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>603</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="basalEdit">
   <property name="geometry">
    <rect>
     <x>603</x>
     <y>300</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="correctionEdit">
   <property name="geometry">
    <rect>
     <x>603</x>
     <y>350</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="carbsEdit">
   <property name="geometry">
    <rect>
     <x>603</x>
     <y>400</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="targetEdit">
   <property name="geometry">
    <rect>
     <x>603</x>
     <y>450</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="existingLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Existing Profiles:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>540</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QLabel" name="profilesLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
